Headset.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Headset
+Comment=Native YouTube Player
+GenericName=Headset
+Exec=Headset %U
+Icon=Headset
```

Comment:
Yes, it is possible but it's not necessary. I use Ubuntu and I don't have the missing icon problem. Currently, the .deb package that electron is creating is working properly without problems.
As long as there's an icon named Headset.(extension) on /usr/share/pixmaps/ then you don't need an absolute path. Currently, this is the case, so there's no problem. I saw issue #35 but I can't replicate this at all. The notification shows the current track as the icon. Headset shows with the correct icon. I'll comment there to check with that user.

About the hook for the .deb package. Electron already does that when creating the package, it does go to the proper location without any problems. I uploaded this file because you requested it, but I think it would be better to setup the parameters when using electron-installer-debian.
You can see the options here: https://github.com/unindented/electron-installer-debian
So, for the `options.categories` you can put the ones from this file. You can also specify the icon using `options.icon` but I think you are already using this because it's working.
The only thing I notice is that electron-installer doesn't have the option for keywords. Since all the keywords are also found under category, this is not a problem to ignore keywords. The only one missing would be stream (which I'm not sure you want your app related to that keyword), and if you want it, you can just include it under the comment, something like `Comment=Native Youtube Player streaming music client`